onnxruntime/core/providers/cpu/tensor/upsample.h

```diff
@@ -59,6 +59,10 @@ class UpsampleBase {
     cubic_coeff_a_ = info.GetAttrOrDefault<float>("cubic_coeff_a", -0.75f);
     exclude_outside_ = info.GetAttrOrDefault<int64_t>("exclude_outside", 0) == 0 ? false : true;
 
+    if (exclude_outside_ == 1 && mode_ != CUBIC) {
```

Comment:
>if (exclude_outside_ == 1 && mode_ != CUBIC) { [](start = 4, length = 46)

I am reading the spec and I am not seeing that exclude_outside is CUBIC mode only

Reply:
the spec does not clarify how this can be applied to other modes... There is no proper documentation available for this either.... I am still trying to find whether this can even be applied to other modes.... onnx tests are also only targeting Cubic mode... that is why I added an error message to catch this until we are in a position to implement this.